IdokladSdk.UnitTests/Tests/Validation/Detailed/ModelValidatorTests.cs

```diff
@@ -887,6 +887,60 @@ public void ModelDateGreaterOrEqualThanAnotherDate_InvalidModel_ReturnsExpectedR
             AssertIsNotValid(result, nameof(model.DateOfExpiration), typeof(DateGreaterThanOrEqualThanAnotherDateAttribute), ValidationType.DateGreaterOrEqualThanAnotherDate);
         }
 
+        [Test]
```

Comment:
Nice to have: Vím, že ty testy na atributy dáváme všechny do jedné testovací třídy, ale já bych vytvořil DateGreaterOrEqualThanAnotherDateTests a tam dal tu tvoji logiku. Už sám jsem nad tím přemýšlel, ale byl jsem líný. Když teď vidím, jak to bobtná, myslím, že je čas to začít dávat do vlastních testovacích tříd :D

Reply:
presuniem do partial class